### ISSUE TITLE
about training_mask and loss function

### ISSUE THREAD

**Issue by LouisRuan** [argman/EAST#110]
What's the use to parameter training_mask in loss function?
after function shrink_poly,score_map is  shrinked
https://github.com/argman/EAST/blob/12d3a6b5c566c8ea231a8c31d2b4a1332ac6482d/icdar.py#L179
in loss function:
https://github.com/argman/EAST/blob/12d3a6b5c566c8ea231a8c31d2b4a1332ac6482d/model.py#L133
y_true_cls(score_map)*training_mask always equal 0, the model_loss always close to 0.01(dice_coefficient result close to 1),after remove training_mask in loss function,my loss value become normal

**Comment by argman** [comment]
training mask is to ignore some small text regions when computing loss

**Comment by caoyangcr7** [comment]
@LouisRuan hello, you mean that just only remove the training_mask in loss functions? I removed the training_mask, but the loss value still always close to 0.01, and never smaller than 0.01....is there any other operations? thanks

**Comment by jiangmiaotong** [comment]
@argman @caoyangcr7  I met the same issue, have you found any reason or solution?

**Comment by caoyangcr7** [comment]
@jiangmiaotong maybe you can refer to https://github.com/argman/EAST/issues/228. May that help you
